rust/signed_doc/src/doc_types/mod.rs

```diff
@@ -1,8 +1,12 @@
 //! An implementation of different defined document types
 //! <https://input-output-hk.github.io/catalyst-libs/architecture/08_concepts/signed_doc/types/>
 
+use std::sync::LazyLock;
+
 use catalyst_types::uuid::Uuid;
 
+use crate::DocType;
+
 /// Proposal document `UuidV4` type.
 pub const PROPOSAL_DOCUMENT_UUID_TYPE: Uuid =
```

Comment:
Hide all these definitions under the `pub mod deprecated`.
But also distinguish from this list a `UUID` types like (which are non `deprecated` and recent one):
- `SUBMISSION_ACTION`
- `PROPOSAL_UUID_TYPE  = PROPOSAL_DOCUMENT_UUID_TYPE`
- `COMMENT_UUID_TYPE  = COMMENT_DOCUMENT_UUID_TYPE`
- `COMMENT_UUID_TYPE  = COMMENT_DOCUMENT_UUID_TYPE`
- `ACTION_UUID_TYPE = PROPOSAL_ACTION_DOCUMENT_UUID_TYPE`